upgrade/Upgrade_Only_CSM_with_iuf.md

```diff
@@ -13,13 +13,12 @@ section of the [Upgrade CSM and additional products with IUF](../operations/iuf/
 
 ## Upgrade Procedure
 
-1. CSM preparation
-
-   Read the _Important Notes_ section of the
+1. Read the _Important Notes_ section of the
    [CSM 1.5.0 or later to 1.6.0 Upgrade Process](Upgrade_Management_Nodes_and_CSM_Services.md)
-   documentation and then follow only these CSM instructions in order:
+   documentation.
 
-   1. [Prepare for Upgrade](prepare_for_upgrade.md)
+1. [Prepare for Upgrade to Next CSM Major Version](https://github.com/Cray-HPE/docs-csm/tree/release/1.6/upgrade/Prepare_for_Upgrade_to_Next_CSM_Major_Version.md)
```

Comment:
I am not sure if referring `.md` files via https link will work nicely with HTML conversion. Most likely it will be a reference to `.md` from HTML page. Any chance it can be made a generic link in the same branch and not specifically in `release/1.6`? In this cse it can be a relative reference, which is processed by HTML converter.